universal-application-tool-0.0.1/app/modules/SecurityModule.java

```diff
@@ -4,50 +4,44 @@
 import static play.mvc.Results.forbidden;
 import static play.mvc.Results.redirect;
 
+import auth.AdminAuthClient;
+import auth.ApplicantAuthClient;
+import auth.AuthIdentityProviderName;
 import auth.Authorizers;
 import auth.CiviFormProfileData;
 import auth.FakeAdminClient;
 import auth.GuestClient;
 import auth.ProfileFactory;
 import auth.Roles;
-import auth.oidc.AdOidcClient;
-import auth.oidc.AdfsProfileAdapter;
-import auth.oidc.IdcsOidcClient;
-import auth.oidc.IdcsProfileAdapter;
-import auth.saml.LoginRadiusSamlClient;
-import auth.saml.SamlCiviFormProfileAdapter;
+import auth.oidc.AdOidcProvider;
+import auth.oidc.IdcsOidcProvider;
+import auth.saml.LoginRadiusSamlProvider;
 import com.google.common.collect.ImmutableMap;
 import com.google.inject.AbstractModule;
+import com.google.inject.ConfigurationException;
 import com.google.inject.Provides;
 import com.google.inject.Singleton;
 import controllers.routes;
 import java.net.URI;
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
 import java.util.Random;
-import javax.annotation.Nullable;
-import javax.inject.Provider;
 import org.pac4j.core.authorization.authorizer.RequireAllRolesAuthorizer;
 import org.pac4j.core.authorization.authorizer.RequireAnyRoleAuthorizer;
 import org.pac4j.core.client.Client;
 import org.pac4j.core.client.Clients;
+import org.pac4j.core.client.IndirectClient;
 import org.pac4j.core.config.Config;
 import org.pac4j.core.context.HttpConstants;
 import org.pac4j.core.context.session.SessionStore;
-import org.pac4j.core.http.callback.PathParameterCallbackUrlResolver;
-import org.pac4j.oidc.client.OidcClient;
-import org.pac4j.oidc.config.OidcConfiguration;
 import org.pac4j.play.CallbackController;
 import org.pac4j.play.LogoutController;
 import org.pac4j.play.http.PlayHttpActionAdapter;
 import org.pac4j.play.store.PlayCookieSessionStore;
 import org.pac4j.play.store.ShiroAesDataEncrypter;
-import org.pac4j.saml.client.SAML2Client;
-import org.pac4j.saml.config.SAML2Configuration;
+import org.springframework.lang.Nullable;
```

Comment:
question (blocking): did you mean to switch from using `javax.annotation.Nullable` to `org.springframework.lang.Nullable`?

Reply:
No I definitely did not thank you!